fakepkg

```diff
@@ -31,6 +31,8 @@ function fakebuild() {
 	PKG_NAME="${1}-${PKG_VER}"
 	PKG_DB_ENTRY="${ROOT_DIR:-/}${db_path}/local/${PKG_NAME}"
 	PKGEXT="${PKGEXT:-.pkg.tar.zst}" # implicitly sets the compression format
+        local _MTREE="${PKG_DB_ENTRY#/}/mtree"
```

Comment:
I am currently in the process of merging this. Can you elaborate a little as to why the replacement of the first forward slash is necessary (`#/`)?

Reply:
I don't remember :laughing:

I should re-install the old version to check.